Source/RxSwift/mergeWith.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  mergeWith.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 12.05.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension Observable {
+    /**
+     Merges `self` with another observable.
```

Comment:
Same here: `Merges elements from the observable sequence with those of a different observable sequence into a single observable sequence.`